utils/README.md

```diff
@@ -0,0 +1,37 @@
+# Running Tests (NEW)
+
+Run the command:
+
+    make ci
+
+to run an array of unittests locally.
+
+# Running From Source
+
+You will need to setup a **virtualenv** to run from source:
+
+    $ virtualenv oo-install
+    $ source ./oo-install/bin/activate
+    $ virtualenv --relocatable ./oo-install/
+    $ python setup.py install
```

Comment:
Virtualenv's are pretty standard. I have no idea what the 'develop' thing
is. I am pulling this from projects I've worked on in the past where we
developed this testing framework. It worked quite well.

On Tue, Aug 23, 2016 at 1:04 PM, Jason DeTiberus <notifications@github.com>
wrote:

> In utils/README.md
> <https://github.com/openshift/openshift-ansible/pull/2341#discussion_r75940102>
> :
>
> > +# Running Tests (NEW)
> > +
> > +Run the command:
> > +
> > +    make ci
> > +
> > +to run an array of unittests locally.
> > +
> > +# Running From Source
> > +
> > +You will need to setup a **virtualenv** to run from source:
> > +
> > +    $ virtualenv oo-install
> > +    $ source ./oo-install/bin/activate
> > +    $ virtualenv --relocatable ./oo-install/
> > +    $ python setup.py install
>
> does
>
> virtualenv --relocatable ./oo-install
> python steup.py install
>
> provide significant benefit over just doing python setup.py develop?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/2341/files/7bde77d8af603232a24019d947458c366b5ca98f#r75940102>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACBYXfwNcQUbivHiVL3DSKR2KIzERbdks5qi1I5gaJpZM4JqLOq>
> .
>



-- 
Tim Bielawa, Software Engineer [ED-C137]
Cell: 919.332.6411  | IRC: tbielawa (#openshift)
1BA0 4FAB 4C13 FBA0 A036  4958 AD05 E75E 0333 AE37